locations/spiders/taco_palenque.py

```
@@ -1,40 +1,60 @@
+        nuxt_data = response.xpath("//script[starts-with(text(), 'window.__NUXT__')]/text()").get()
+
+        parameter_names = re.search(r"function\(([\w_$,]+)\)", nuxt_data).group(1).split(",")
+        begin_parameter_values = nuxt_data.rfind("}}}}(") + 5
+        parameter_values = chompjs.parse_js_object("[" + nuxt_data[begin_parameter_values : nuxt_data.rfind(")")] + "]")
+
+        # Locations are grouped by state and then by city, so gather each of the per-city arrays.
+        locations = []
+        for city in re.finditer(r"locations:\[", nuxt_data[:begin_parameter_values]):
+            locations.extend(
+                resolve_parameters(
+                    chompjs.parse_js_object(nuxt_data[city.end() - 1 :]), dict(zip(parameter_names, parameter_values))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)